.editorconfig

```diff
@@ -125,11 +125,15 @@ csharp_style_pattern_matching_over_is_with_cast_check = true:suggestion
 csharp_style_pattern_matching_over_as_with_null_check = true:suggestion
 csharp_style_prefer_not_pattern = true:suggestion
 csharp_style_prefer_extended_property_pattern = true:suggestion
-csharp_style_var_for_built_in_types = false:error
-csharp_style_var_when_type_is_apparent = false:error
-csharp_style_var_elsewhere = false:error
+
+csharp_style_var_for_built_in_types = false:silent
+csharp_style_var_when_type_is_apparent = false:silent
+csharp_style_var_elsewhere = false:silent
 
 # CA2249: Consider using 'string.Contains' instead of 'string.IndexOf'
 dotnet_diagnostic.CA2249.severity = error
+csharp_space_around_binary_operators = before_and_after
+
+# IDE0008: Use explicit type
+dotnet_diagnostic.IDE0008.severity = silent
```

Comment:
Please explain the addition of these rules.

Reply:
made the changes! Use Explicit Types was disabled somehow, i updated the rule as an error and also made the changes to the code